src/NuGet.Core/NuGet.Build.Tasks/Common/MSBuildLogger.cs

```diff
@@ -1,4 +1,7 @@
-﻿using Microsoft.Build.Framework;
+﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Just now realized how many file headers were missing...that's hilarious, I'll move it out haha